### PR TITLE
Telegraf 1.20.4

### LINKED ISSUE #3415: Telegraf 1.20.4
https://github.com/influxdata/telegraf/blob/master/CHANGELOG.md

### PR DESCRIPTION
**Pull request by noramullen1**
Closes #3415

### CODE REVIEW

File: content/telegraf/v1.20/about_the_project/release-notes-changelog.md [hunk 1]
```diff
@@ -9,6 +9,29 @@ menu:
     weight: 10
     parent: About the project
 ---
+## v1.20.4 [2021-11-17]
+
+- Update BurntSushi/toml from 0.3.1 to 0.4.1.
+- Update gosnmp module from 1.32 to 1.33.
+- Update go.opentelemetry.io/otel from v0.23.0 to v0.24.0.
+- Linter fixes for plugins h-o (contributed by [@zak-pawel](https://github.com/zak-pawel)).
```
Comment (sanderson) [inline content/telegraf/v1.20/about_the_project/release-notes-changelog.md:17]:
```suggestion
- Update `BurntSushi/toml` from 0.3.1 to 0.4.1.
- Update `gosnmp` module from 1.32 to 1.33.
- Update `go.opentelemetry.io/otel` from v0.23.0 to v0.24.0.
- Fix plugin linters.
```

File: content/telegraf/v1.20/about_the_project/release-notes-changelog.md [hunk 2]
```diff
@@ -9,6 +9,29 @@ menu:
...
+
+### Input plugin updates
+- Cisco Model-Driven Telemetry (`cisco_telemetry_mdt`): Move to new protobuf library.
+- InfluxDB (`influxdb`): Update input schema docs.
+- Intel RDT (`intel_rdt`): Correct the timezone to use local timezone by default instead of UTC from metrics gathered from the `pqos` tool.
+- IPMI Sensor (`ipmi`): Passwords will not be stored in log files and now display `REDACTED` text to maintain security.
```
Comment (sanderson) [inline content/telegraf/v1.20/about_the_project/release-notes-changelog.md:23]:
```suggestion
- IPMI Sensor (`ipmi`): Redact passwords in log files to maintain security.
```

File: content/telegraf/v1.20/about_the_project/release-notes-changelog.md [hunk 3]
```diff
@@ -9,6 +9,29 @@ menu:
...
+- Modbus (`modbus`): Do not build on OpenBSD.
+- MySQL (`mysql`):
+  - Fix type conversion follow-up.
+  - Correctly set the default paths.
+- NVIDIA SMI (`nvidia_smi`): Correctly set the default paths.
+- Proxmox (`proxmox`): Parse the column types of the server status.
+- SQL Server (`sqlserver`): Add elastic pool in supported versions.
+
+### Output plugin updates
+- Loki (`loki`): Includes the metric name as a label for improved query performance and metric filtering.
```
Comment (sanderson) [inline content/telegraf/v1.20/about_the_project/release-notes-changelog.md:33]:
```suggestion
- Loki (`loki`): Include the metric name as a label for improved query performance and metric filtering.
```

**Review by sanderson — APPROVED** [review]
It added Michelle as the review, better get that fixed 😄 . Just some minor suggestions.

**Review by sjwang90 — APPROVED** [review]

**Pull request merged**